roles/openstack_create_server/tasks/main.yml

```diff
@@ -151,18 +151,10 @@
     openstack_project: "{{ project_name.stdout }}"
   when: openstack_project == "" and project_name.stdout != ""
 
-# Raise the quotas from the defaults if possible.
+# Change the quotas from the defaults for the project.
 - name: Setting the quotas for {{ openstack_project }}
   shell: "{{ openstack }} quota set --{{ item }} {{ quotas[ item ] }} {{ openstack_project }}"
-  with_items:
-    - cores
-    - floating-ips
-    - gigabytes
-    - instances
-    - ports
-    - ram
-    - secgroups
-    - volumes
+  with_items: "{{ quotas }}"
```

Comment:
I my opinion using a single command with all of the quotas in a single command would be nicer because it would occur in one command rather than say ~24 commands each with say a run-time of 2-3s.

Reply:
Thanks for the review @akrzos . I do appreciate this would be faster, but I have tested this code on an undercloud already and I don't know how I would convert this to one command at this time with Ansible.